GargasMAOverview.md

```diff
@@ -0,0 +1,41 @@
+﻿# The Acoustics of Smiling Voice
+
+## **Abstract**
+
+## **Introduction**
+
+General information about the aim of the thesis, introduction to the subject of the thesis, information about its structure and what would be presented in the following chapters.
+
+## **Chapter 1. Theory**
+
+1.Phonetics
+1.1 Articulatory Phonetics
+1.1.1 Segmental Phonetics
+
+1.1.2 Prosody
+
+1.2 Acoustic Phonetics
+
+2. Smile and emotions
+
+3. Smiling Voice
+
+3.1 Production of Smiling Voice
+
+## **Chapter 2. Methodology**
+
+The collected data consists of forty recordings of my own voice. Half of the voice samples are in Polish and the second half in English. The data includes only short words including vowels. Among all of the samples, there occur all vowels existing in the Polish and English language. The recordings can be also divided into two groups - ones recorded during smiling and during the normal speech. With the use of Praat, I would like to measure such parameters as pitch, intensity, formants (F1, F2) to check the differences between words articulated with and without a smile and to find out whether you can hear the smile.  To facilitate and shorten the length of obtaining the aforementioned parameters, I will use the script "'editor $ ''tab $ ''f1:2''tab $ ''f2:2''tab $ ''f3:2''tab $ ''f0:2''tab $ ''intensity:2''tab $' 'dur:2'" that enables me to receive all the parameters without measuring them on my own.
+Example output:
+![Spectograph](https://i.pinimg.com/564x/67/ab/7a/67ab7afb525f9513660fd66a5248f761.jpg)
+
+## **Chapter 3. Analysis**
+
+Analyzing the spectrographs and investigating whether the speaker was smiling or not and marking the differences between the Smiling Voice and Non-smiling one.
+Here I provide a spectrograph image of the word "Poniedziałek":
+![enter image description here](https://i.pinimg.com/564x/ca/a7/c8/caa7c83862bcc7ae66b4fcc5f861ed54.jpg)
```

Comment:
1. Don't leave the "enter image description here" placeholder; replace it with a real title
2. Put some space after the image. Look at how it's currently rendered...?